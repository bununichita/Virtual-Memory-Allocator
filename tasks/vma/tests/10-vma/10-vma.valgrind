==49288== Memcheck, a memory error detector
==49288== Copyright (C) 2002-2017, and GNU GPL'd, by Julian Seward et al.
==49288== Using Valgrind-3.18.1 and LibVEX; rerun with -h for copyright info
==49288== Command: /home/nicky/Desktop/SDA/Virtual-Memory-Allocator/vma
==49288== Parent PID: 49287
==49288== 
==49288== 
==49288== HEAP SUMMARY:
==49288==     in use at exit: 0 bytes in 0 blocks
==49288==   total heap usage: 10 allocs, 10 frees, 8,377 bytes allocated
==49288== 
==49288== All heap blocks were freed -- no leaks are possible
==49288== 
==49288== For lists of detected and suppressed errors, rerun with: -s
==49288== ERROR SUMMARY: 0 errors from 0 contexts (suppressed: 0 from 0)
