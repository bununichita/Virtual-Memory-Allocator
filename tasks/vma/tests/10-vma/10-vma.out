This zone was already allocated.
Total memory: 14 bytes
Free memory: 10 bytes
Number of allocated blocks: 1
Number of allocated miniblocks: 1

Block 1 begin
Zone: 5 - F
Miniblock 1:	5	-	F	| RW-
Block 1 end
